### PR TITLE
Add EatData and ChibiEatData getters/setters/enums

### PR DESCRIPTION
**Pull request by mrbengtsson**

### CODE REVIEW

File: actor/ChibiEatData.h [hunk 1]
```diff
@@ -25,18 +40,57 @@ class ChibiEatData  // vtbl Address: 0x1000145C
...
+    {
+        return mScale;
+    }
+
+    const sead::Vector3f& getScale() const
+    {
+        return mScale;
+    }
+
+    void setScale(const sead::Vector3f& scale)
```
Comment (aboood40091) [inline actor/ChibiEatData.h:73]: I don't see the point of a `setScale` function if there's a non-const `getScale` variant.

File: actor/EatData.h [hunk 2]
```diff
@@ -35,12 +55,57 @@ class EatData   // vtbl Address: 0x10001594
...
+    {
+        return mScale;
+    }
+
+    const sead::Vector3f& getScale() const
+    {
+        return mScale;
+    }
+
+    void setScale(const sead::Vector3f& scale)
```
Comment (aboood40091) [inline actor/EatData.h:88]: Same as the other comment.

**Review by aboood40091 — CHANGES_REQUESTED** [review]

File: actor/ChibiEatData.h (hunk 1 above)
Comment (mrbengtsson) [reply actor/ChibiEatData.h:73]: You can remove them after you merge the pull request :)

Comment (aboood40091) [reply actor/ChibiEatData.h:73]: I would've preferred that to be part of the pull request hence the change request, but oh well.

**Pull request merged**